radio/src/targets/common/arm/stm32/heartbeat_driver.cpp

```diff
@@ -42,7 +42,13 @@ static void trigger_intmodule_heartbeat()
 #endif
 
   mixerSchedulerResetTimer();
-  mixerSchedulerISRTrigger();
+
+  // Generate a timer update event (TIM_EGR_UG) to reload the Prescaler and the repetition 
+  // counter value immediately to avoid making FreeRTOS calls within this ISR:
+  // - fires MIXER_SCHEDULER_TIMER interrupt after returning from this ISR
+  // - MIXER_SCHEDULER_TIMER_IRQHandler(void) takes care of making FreeRTOS calls
+  //   to ensure switching to highest priority task.
+  MIXER_SCHEDULER_TIMER->EGR = TIM_EGR_UG;  
```

Comment:
Can I please have a method in `mixer_scheduler_driver.cpp` that does just that please? (something like `mixerSchedulerSoftTrigger()`)

Reply:
done